tests/test_distributedarray.py

```diff
@@ -118,14 +132,22 @@ def test_distributed_math(par1, par2):
 
 
 @pytest.mark.mpi(min_size=2)
-@pytest.mark.parametrize("par1, par2", [(par3_1, par3_2), (par4_1, par4_2)])
+@pytest.mark.parametrize("par1, par2", [(par6, par7), (par8, par9)])
 def test_distributed_dot(par1, par2):
     """Test Distributed Dot product"""
-    pass
+    arr1 = DistributedArray.to_dist(x=par1['x'], partition=par1['partition'], axis=par1['axis'])
+    arr2 = DistributedArray.to_dist(x=par2['x'], partition=par2['partition'], axis=par2['axis'])
+    assert_almost_equal(arr1.dot(arr2), np.dot(par1['x'].flatten(), par2['x'].flatten()), decimal=3)
```

Comment:
Could you please change this also? Thanks.

Reply:
Done :+1: